crypto/src/main/kotlin/web5/sdk/crypto/InMemoryKeyManager.kt

```diff
@@ -90,9 +90,6 @@ public class InMemoryKeyManager : KeyManager {
    * Imports [jwk] and returns the alias that refers to it.
    */
   public fun import(jwk: JWK): String {
-    require(jwk.isPrivate) {
-      "Importing a non-private key is not permitted"
-    }
```

Comment:
what's the use-case driving the ability to import a public key?

Reply:
I wanted to use  the same test vectors from the `ion-sdk` repository. The update request tests uses the public key of the next update, but not the private key. 

See https://github.com/TBD54566975/web5-kt/blob/ad6a91e1a8cac29c752156116fdf253395c5bf7c/dids/src/test/kotlin/web5/sdk/dids/DidIonTest.kt#L292 for where it's used. 

And see https://github.com/decentralized-identity/ion-sdk/blob/main/tests/IonRequest.spec.ts#L48 for the original test.